common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -5055,6 +5064,14 @@ object CelebornConf extends Logging {
       .checkValue(v => v > 0.0 && v <= 1.0, "Value must be between 0 and 1 (inclusive)")
       .createWithDefault(0.4)
 
+  val CLIENT_APPLICATION_UUID_SUFFIX_ENABLED: ConfigEntry[Boolean] =
+    buildConf("celeborn.client.application.uuidSuffix.enabled")
+      .categories("client")
+      .version("0.6.0")
+      .doc("Whether to add UUID suffix for application id for unique. When `true`, add UUID suffix for unique application id.")
```

Comment:
This configuration only takes effect when using the Spark and MR engines, so we must specify this in doc.